src/resources/resources.json

```diff
@@ -1853,6 +1849,20 @@
       "de": "Aktualisiert"
     }
   },
+  "recentContributions": {
+    "pageSubtitle": {
+      "en": "Open contributions made in the last 3 months.",
+      "de": "Open contributions made in the last 3 months."
```

Comment:
Help needed here as well please.

Reply:
Öffentliche Beiträge in den letzten 3 Monaten (maybe also "Public" instead of "Open" in English?)

Reply:
Right, changed both!